### DISCUSSION TITLE
pimcore.sock file is not created

### DISCUSSION THREAD

**Discussion by MoorthyRamasamy** [pimcore/pimcore#13740] [General]
i have installed the pimcore by using the following command 
COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/demo my-project
after the installation configured the nginx as per the instruction provided 
when i tried to access the index.php i am not getting the page as  follows 

![image](https://user-images.githubusercontent.com/49427828/205703442-c98975f5-f3b1-4d4e-9cb2-71b034ae500e.png)

and also i observed pimcore.sock file not created under the /var/run/php/pimcore.sock

can you please help us to resolve the issue

**Comment by mattamon** [comment]
Hey  @MoorthyRamasamy,

did you follow all steps which are described [here](https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html)?

Did you create the database and install Pimcore?
```bash
cd ./my-project
./vendor/bin/pimcore-install
```

Can you please share your server setup?
